transport/grpc/doc.go

```diff
@@ -40,6 +40,30 @@
 //     Inbounds: yarpc.Inbounds{myInbound},
 //   })
 //
+// To configure TLS on your service listener, pass
+// credentials.TransportCredentials as an InboundCredentials InboundOption.
+// There are various ways to create credentials.TransportCredentials. See
+// https://godoc.org/google.golang.org/grpc/credentials#TransportCredentials.
+//
+//   listener, err := net.Listen("tcp", ":4443")
+//   if err != nil {
+//     return err
+//   }
+//
+//   myTLSConfig := &tls.Config{
+//     // any arbitrary valid tls.Config
+//   }
+//   myTransportCredentials := credentials.NewTLS(myTLSConfig)
+//   myInboundCredentials := InboundCredentials(myTransportCredentials)
```

Comment:
nit: you typically don't put the actual option into a variable since there are
going to be possibly an unknown number of them. The options pattern naturally
provides some named-parameters-like readability:

    NewThing(foo,
        EnableFoo(true),
        AllowUsers("foo", "bar", "baz"),
        AuthenticateWith(credentials),
    )

Reply:
done